include/open62541/plugin/pubsub.h

```diff
@@ -47,6 +48,9 @@ struct UA_PubSubChannel {
     UA_PubSubChannelState state;
     UA_PubSubConnectionConfig *connectionConfig; /* link to parent connection config */
     UA_SOCKET sockfd;
+#ifdef UA_ENABLE_PUBSUB_ETH_UADP_XDP
```

Comment:
This should be part of the XDP-enabled PubSub plugin only.
And not part of the public header.
See the discussion we had for the ETF feature.